### ISSUE TITLE
Variable removed from coffee-mode, causing literate-coffee-mode to barf

### ISSUE THREAD

**Issue by zot** [syohex/emacs-literate-coffee-mode#9]
coffee-string-interpolation-regexp has been removed, so I put this def in my setup.  Maybe add it or make a local one?
``` elisp
(defvar coffee-string-interpolation-regexp "#{[^}\n\\\\]*\\(?:\\\\.[^}\n\\\\]*\\)*}")
```

**Comment by syohex** [comment]
@zot Thanks for reporting issue. Sorry I didn't notice this long time. I have fixed at #10. Please check latest version.

**Comment by zot** [comment]
@syohex thanks for pushing the fix out so quickly!
